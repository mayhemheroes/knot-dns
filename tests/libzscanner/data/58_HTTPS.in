$ORIGIN	.
$TTL	1

; HTTPS is the same as SVCB, so there are the differences and basics only.

; OK
@	HTTPS	0	.
@	HTTPS	65535	@	; Comment
@       HTTPS	1	.       key65535="a" key1000=b port=4 key7 mandatory=key7,port
@       TYPE65	\# 3 000000
@       TYPE65	0	.
@       https	0	.

; KO
@	HTTPS
@	HTTPS						; Empty rdata
@	HTTPS	\# 0					; Hex empty rdata
@	HTTPS	65536	.				; Priority overflow
@	HTTPS	0					; Missing item
