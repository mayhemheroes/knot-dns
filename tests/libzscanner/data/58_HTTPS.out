OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0041
RDATA=000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0041
RDATA=FFFF00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0041
RDATA=00010000000004000300070003000200040007000003E8000162FFFF000161
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0041
RDATA=000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0041
RDATA=000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0041
RDATA=000000
------
WARNG=ZS_BAD_NUMBER
------
WARNG=ZS_BAD_NUMBER
------
WARNG=ZS_BAD_HEX_CHAR
------
WARNG=ZS_NUMBER16_OVERFLOW
------
WARNG=ZS_BAD_RDATA
------
